City-of-Helsinki/berth-reservations-admin
# VEN-1241 | New Invoice Card

## Description :sparkles:
- Wrap all invoicing supported actions and states in a custom hook and use it on both the application and the customer pages
- Implement a new invoice card
- Support actions on multiple offers

## Issues :bug:

### Closes :no_good_woman:
[VEN-1241](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1241): New invoices card for single customer view

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️

### Manual testing :construction_worker_man:
- Go to a single customer page
- Under "Laskut" card you will see all orders a customer has made
- From the dropdown above the card you can perform actions on the selected table rows

**Note:** The selected action will get disabled if at least **one** of the selected orders (rows) doesn't satisfy the conditions of the action in question 

## Screenshots :camera_flash:
![image](https://user-images.githubusercontent.com/23040926/116388777-dd4c8280-a824-11eb-8fd0-965619a8de44.png)

## Additional notes :spiral_notepad:
